core/queryparser/sparql/src/main/java/org/eclipse/rdf4j/query/parser/sparql/ast/SyntaxTreeBuilderConstants.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 Eclipse RDF4J contributors, Aduna, and others.
+ * Copyright (c) 2020 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
No big dealm but there's no need to update copyright years in existing files. We never update these: they reflect the year the file was first created, not when it was last edited.

Reply:
Updated the headers fro the existing files e.g. changed back to `2018` from `2020` ...